cs-office-365/src/main/java/io/cloudslang/content/office365/utils/HttpUtils.java

```diff
@@ -160,4 +160,47 @@ private static String getLoginType(@NotNull final String userPrincipalName,
         return userId;
     }
 
+    @NotNull
+    public static String createMessagesPath(@NotNull final String userPrincipalName,
```

Comment:
Shouldn't these methods createMessagePath and createMessagesPath have a more descriptive name?